### ISSUE TITLE
Unofficial builds for Fedora now available

### ISSUE THREAD

**Issue by bugzy** [keepassxreboot/keepassxreboot.github.io#7]
Just wanted to point out that unofficial builds for fedora are available at:
https://copr.fedorainfracloud.org/coprs/bugzy/keepassxc/

so that it can be reflected on https://keepassxc.org/download#linux

**Comment by phoerious** [comment]
Thanks for taking the trouble making those packages. I had a quick look through the source packages. A few things I noticed:

- you are building with debug symbols (is that intended?)
- you are not building autotype
- you are building tests (not needed for production)

**Comment by bugzy** [comment]
I have modified the spec file according to the items that you pointed out above. I am now building with:

       -DCMAKE_INSTALL_PREFIX=/usr \
        -DCMAKE_VERBOSE_MAKEFILE=OFF \
        -DCMAKE_BUILD_TYPE=RelWithDebInfo \
        -DWITH_XC_YUBIKEY=ON \
        -DWITH_TESTS=OFF \
        -DWITH_XC_HTTP=ON \
        -DWITH_XC_AUTOTYPE=ON

**Comment by bugzy** [comment]
I see the issue now. I have modified the spec file to build with:
-DCMAKE_INSTALL_PREFIX=/usr \
-DCMAKE_VERBOSE_MAKEFILE=OFF \
-DWITH_XC_YUBIKEY=ON \
-DWITH_TESTS=OFF \
-DWITH_XC_HTTP=ON \
-DWITH_XC_AUTOTYPE=ON

(removed debug symbols)
This should address all three items that you mentioned in your original comment:
-I was building with debug symbols unintentionally (however, I suppose building with debug symbols on would help with diagnosing crashes if and when they occur)
- I am now building with autotype support
- Tests are now turned off

Let me know if you spot any other issues. 
Thanks

**Comment by phoerious** [comment]
You should build without the Yubikey flag. Yubikey support has not been merged yet, but the flag will cause the Yubikey plugin to be listed on the "About" dialog, nevertheless.

**Comment by bugzy** [comment]
Got it. Yubikey flag removed and new build is up. Thanks again

**Comment by phoerious** [comment]
Thanks a lot. I added the link to the website.

**Comment by psoberoi** [comment]
Where can I file bugs about the fedora package? I'm having trouble installing both keepassx and keepassxc together:

$ sudo dnf copr enable bugzy/keepassxc

$ sudo dnf install keepassxc
...
Running transaction check
Transaction check succeeded.
Running transaction test
The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing 'dnf clean packages'.
Error: Transaction check error:
file /usr/share/mimelnk/application/x-keepass2.desktop from install of keepassxc-2.1.1-4.fc25.x86_64 conflicts with file from package keepassx-1:2.0.3-2.fc25.x86_64

As along-time keepassx user, I want to be able to test out keepassxc before uninstalling keepassx.

Thanks for packaging this!

**Comment by bugzy** [comment]
@psoberoi 
Thanks for pointing out the issue.
I have put up an new build that renames the .desktop file so that there is no conflict.
In the future, you can report issues at https://github.com/bugzy/keepasxc-rpm-spec/issues

**Comment by psoberoi** [comment]
Thanks, I have them both installed in parallel now!
